io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -124,14 +124,14 @@ public function channelWriteString(WritableChannel writableChannel, string conte
     }
 }
 
-# Write an array of lines to a channel.
+# Write an array of lines to a channel
```

Comment:
This accepts a `string[]` array. When writing to the channel, does this add a new line char after writing each element in the array? Or is this just simply an array of strings being written to the channel?

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8